base/_buttons.scss

```diff
@@ -0,0 +1,63 @@
+@charset "UTF-8";
+
+// buttons.scss
+// This file contains all styles related to the button element.
+// see http://bitters.bourbon.io
+// dependent on utilities/color.scss
+// -----------------------------------------------------------------------------
+
+
+// Variables in use on this file.
+// Variables are defined with !default flags, and can be overwritten in
+// `utilities/_variables.scss`
+// -----------------------------------------------------------------------------
+$action-color: color(generic, blue) !default;
```

Comment:
Code climate is giving the error: "Color should be written in hexadecimal form"
I think we can disable that check. Any objections?